docs/quickstarts/ios.mdx

```diff
@@ -0,0 +1,296 @@
+---
+description: Add authentication and user management to your iOS app with Clerk.
+---
+
+# iOS Quickstart (Beta)
+
+<Callout type="warning">
+  The Clerk iOS SDK is currently in beta. It is **not yet recommended for production use**.
+</Callout>
```

Comment:
`<Callout>` is deprecated in favour of [blockquote-based syntax](https://github.com/clerk/clerk-docs/blob/main/CONTRIBUTING.md#callouts):

```suggestion
> [!WARNING]
> The Clerk iOS SDK is currently in beta. It is **not yet recommended for production use**.
```